README.md

```diff
@@ -60,16 +66,4 @@ Feel free to check the [issues page](../../issues/).
 
```

Comment:
- [ ] OPTIONAL
It's recommended that you add an MIT license, or any other license to your ReadMe. Implementing this will make your ReadMe appear more professional.